### ISSUE TITLE
Save settings individually per game

### ISSUE THREAD

**Issue by spielvan** [flyinghead/flycast#252]
Using the flycast version outside the retroarch, when I set up a game and I have to save "Make game config" elke saves it for all games, but it has a game like Power Stone 2, which needs Force Force ce to fix the graph, but in other games this checked option bugs the game choice menu.

**Comment by flyinghead** [comment]
Are you using `Force Windows CE` to run Power Stone 2?

**Comment by spielvan** [comment]
Power Stone 2 (both dreamcast and Naomi) Without the Force Windows CE option checked, notice the black graphics bug blinking.
![Screenshot_2021-05-24-09-22-32](https://user-images.githubusercontent.com/17939556/119347730-4f708580-bc72-11eb-849f-bb5e1431d4d8.png)


Power Stone 2 (both Dreamcast and Naomi) With the Force Windows CE option checked, without the blinking black graphics bug.

![Screenshot_2021-05-24-09-24-21](https://user-images.githubusercontent.com/17939556/119347842-74fd8f00-bc72-11eb-88e5-d47e9070a6af.png)

This on Box android tv with gpu mali450 Open gl 2.0 ES (Occurs both with the Flycast core retorarch, as well as a separate version of the retroarch).

An interesting thing that in the psx mini classic these graphic bugs do not occur

**Comment by flyinghead** [comment]
Interesting. I can reproduce the same thing on Mali 400.

This is due to a depth setting that is enabled for all Windows CE games. I will enable it for this game as well.

Do you know of other games that need `Force Windows CE` to fix graphics issues?

**Comment by spielvan** [comment]
Interesting. I can reproduce the same thing on Mali 400.

This is due to a depth setting that is enabled for all Windows CE games. I will enable it for this game as well.

Do you know of other games that need Force Windows CE to fix graphics issues?

**Comment by flyinghead** [comment]
This seems to be very specific to the Mali 400/450. Some games are also negatively impacted by this change on Mali. Still working on this.

To answer your original question about per-game settings. Here is the procedure:
1. Start the game with your normal settings.
2. Pause the game
3. In Settings, tap `Make Game Config`
4. Set the configuration specific to this game. For example, enable `Force Windows CE`
5. Done

**Comment by spielvan** [comment]
> This seems to be very specific to the Mali 400/450. Some games are also negatively impacted by this change on Mali. Still working on this.
> 
> To answer your original question about per-game settings. Here is the procedure:
> 
> 1. Start the game with your normal settings.
> 2. Pause the game
> 3. In Settings, tap `Make Game Config`
> 4. Set the configuration specific to this game. For example, enable `Force Windows CE`
> 5. Done
> 
> You might have to restart the game for the settings to take effect. This is the case for `Force Windows CE`

I followed the step by step, but still not saving individually, I'm using the latest build 86d014c

**Comment by flyinghead** [comment]
Can you post your `emu.cfg` file after following these steps?

**Comment by spielvan** [comment]
[T1211N]
config.Dreamcast.ForceWindowsCE = yes
config.aica.AutoLatency = no
config.aica.BufferSize = 750
config.pvr.MaxThreads = 2
config.rend.MaxFilteredTextureSize = 8
input.MouseSensitivity = 80
input.device1.1 = 3

[audio]
backend = auto
disable = 0

[config]
Debug.SerialConsoleEnabled = no
Debug.SerialPTY = no
Dreamcast.AutoSavestate = no
Dreamcast.Broadcast = 0
Dreamcast.Cable = 3
Dreamcast.ContentPath = /storage/udisk0/part1/ROMS/DREAMCAST
Dreamcast.ForceWindowsCE = no
Dreamcast.FullMMU = no
Dreamcast.HideLegacyNaomiRoms = yes
Dreamcast.Language = 1
Dreamcast.Region = 1
Dynarec.Enabled = yes
Dynarec.idleskip = no
Dynarec.safe-mode = yes
UseReios = no
aica.AutoLatency = yes
aica.BufferSize = 5644
aica.DSPEnabled = no
aica.NoSound = no
pvr.AutoSkipFrame = 0
pvr.MaxThreads = 3
pvr.rend = 0
rend.CrossHairColor1 = 0
rend.CrossHairColor2 = 0
rend.CrossHairColor3 = 0
rend.CrossHairColor4 = 0
rend.CustomTextures = no
rend.DelayFrameSwapping = no
rend.DumpTextures = no
rend.ExtraDepthScale = 1.000000
rend.FloatVMUs = no
rend.Fog = yes
rend.MaxFilteredTextureSize = 256
rend.ModifierVolumes = yes
rend.PerStripSorting = no
rend.RenderToTextureBuffer = no
rend.Resolution = 480
rend.Rotate90 = no
rend.ScreenStretching = 100
rend.ShowFPS = no
rend.SuperWideScreen = no
rend.TextureUpscale = 1
rend.TranslucentPolygonDepthMask = no
rend.UseMipmaps = yes
rend.WideScreen = no
rend.WidescreenGameHacks = no
rend.vsync = yes
ta.skip = 0

[input]
MouseSensitivity = 100
VirtualGamepadVibration = 20
device1 = 0
device1.1 = 1
device1.2 = 1
device2 = 10
device2.1 = 10
device2.2 = 10
device3 = 10
device3.1 = 10
device3.2 = 10
device4 = 10
device4.1 = 10
device4.2 = 10
maple_02b1ce58713e0e531eab61b0742baac6893ff49f = 0
maple_3ff0f65e296fcbd89914545764ea53395eb45fc1 = 0
maple_485d69228e24f5e46da1598745890b214130dbc4 = 0
maple_a604f897f8e4cc8e80cbc672b77a7a5b4e7c6b6d = 0
maple_a718a782d34bc767f4689c232d64d527998ea7fd = 0
maple_d2c52ff0f656fac4cd7b7a118d575e0109a9fe1c = 0
maple_dc4619eefd76423d152d1b3789814fb245dfd008 = 0
maple_virtual_gamepad_uid = 0

[network]
ActAsServer = no
DNS = 46.101.91.123
EmulateBBA = no
Enable = no
server = 

[validate]
OpenGlChecks = no

**Comment by flyinghead** [comment]
You're right: Force Windows CE doesn't work as a per-game setting. This is a bug.

**Comment by flyinghead** [comment]
Should be fixed in the latest master build.
